draft-sharabayko-mops-srt.md

```diff
@@ -722,30 +723,30 @@ and bonded SRT connections (group connections).
 
 GroupID: 32 bits.
 : The identifier of a group whose members include the sender socket that is making a connection.
-  The target socket that should interpret it should belong to the corresponding group on its side
-  (or should create one, if it doesn't exist).
+  The target socket that is interpreting GroupID should belong to the corresponding group on the target side.
+  If such a group does not exist, the target socket should create it.
```

Comment:
"Should" -> "must" because it must belong to a group, or consider creating one. Although not 100% sure about the word "must" here, so upon your decision. 🤔 
"Should" -> "may" because it depends e.g. on whether a socket is allowed to accept group connections or not.

```suggestion
  The target socket that is interpreting GroupID must belong to the corresponding group on the target side.
  If such a group does not exist, the target socket may create one.
```